packages/cli/src/Server.ts

```diff
@@ -779,236 +798,259 @@ class App {
 		// Returns workflows
 		this.app.get(
 			`/${this.restEndpoint}/workflows`,
-			ResponseHelper.send(async (req: express.Request, res: express.Response) => {
-				const queryBuilder = Db.collections.Workflow!.createQueryBuilder('w');
-				queryBuilder.select(['w.id', 'w.name', 'w.active', 'w.createdAt', 'w.updatedAt']);
-				queryBuilder.leftJoinAndSelect('w.tags', 't');
-
-				if (req.query.filter) {
-					const jsonFilters = JSON.parse(req.query.filter as string);
-					const keys = Object.keys(jsonFilters);
-					keys.forEach((key) => {
-						queryBuilder.andWhere(`w.${key} = :${key}`, { [key]: jsonFilters[key] });
+			ResponseHelper.send(async (req: WorkflowRequest.GetAll) => {
+				let workflows: WorkflowEntity[] = [];
+
+				if (req.user.globalRole.name === 'owner') {
+					workflows = await Db.collections.Workflow!.find({
+						select: ['id', 'name', 'active', 'createdAt', 'updatedAt'],
+						relations: ['tags'],
+						where: req.query,
```

Comment:
In the codebase, `getWorkflows()` is never called with `filter` at all, so I cannot know how it will be used, but assuming `?a=1&b=2`, then there will be no `filter` field and the query string will already be parsed as `{ a: '1', b: '2' }`.